sw/banshee/src/tran.rs

```diff
@@ -1215,6 +1215,7 @@ impl<'a> InstructionTranslator<'a> {
         match data.op {
             riscv::OpcodeImm12Rs1::Scfgwi => {
                 // ssr write immediate holds address offset in imm12, content in rs1
+                // rs2[11:5]=reg_word rs2[4:0]=dm -> addr_off = {dm, reg_word[4:0], 000}
                 let dm = (imm as u64) & 0x1f;
                 let reg_word = ((imm as u64) >> 5) & 0x7f;
```

Comment:
Don't forget to change the mask here as well. :wink:

Reply:
:+1: 911c1b9